channels/fast-4.4.yaml

```diff
@@ -3,3 +3,6 @@ versions:
 - 4.3.12
 - 4.3.13
 # No 4.3.14 because it was missing version which 4.3.14 can be upgraded. It is rebuilt as 4.3.15
+# No 4.3.15 as we moved to build 4.3.16 to fix https://bugzilla.redhat.com/show_bug.cgi?id=1827337
+# No 4.3.16 because it was rejected by the release controller as ovn container image had unsigned RPMs
+# No 4.3.17 because we moved to 4.3.18 which includes fix for https://bugzilla.redhat.com/show_bug.cgi?id=1828104
```

Comment:
```yaml
# No 4.3.17 because it lacked OCICNI context for robust recovery from network I/O timeouts: https://bugzilla.redhat.com/show_bug.cgi?id=1828104
```